Readme.md

```diff
@@ -10,14 +10,17 @@ New to Docker? See the [Runbook](./RUNBOOK.md) for some useful recipes.
 
 ## Contents
 
-- [Prerequisites](#installation-instructions-for-prerequisites)
-- [Quick Start](#quick-start)
-  - [Initializing the database](#option-1-run-the-web-installer)
-  - [Restoring database from a backup](#option-2-restore-from-backup)
-- [Development](#development)
-  - [Docker Compose Configuration](#compose-configuration)
-  - [Volumes](#volumes)
-- [TODO](#todo)
+- [Metakgp Wiki Setup](#metakgp-wiki-setup)
```

Comment:
Not sure why this is required? Readmes generally contain setup information. Not required IMO 🙂

Reply:
Why is this extra bullet required?

Reply:
This was automatically added based on the content heading inside the README.md. I didnot add it manually, but I feel this is appropriate and we should not change it. It is not conveying anything false.